docs/release-notes/RELEASE-NOTES-0.21.0.md

```diff
@@ -4,6 +4,7 @@
 It:
 * Adds support for type aliases to the Sophia compiler.
 * Changes the target (difficulty) calculation algorithm to use [DigiShield v3][digishield_v3]. This impacts consensus.
+* Fixes fees calculations, adding which has been too generous. This impacts consensus.
```

Comment:
This doesn't read right... "Fixes miner fee reward calculations, was too generous before" perhaps?